### ISSUE TITLE
Taris Sith Base cleanup

### ISSUE THREAD

**Issue by JCarter426** [KOTORCommunityPatches/K1_Community_Patch#149]
A few minor issues I encountered:

~1. Receptionist in commoner clothing. Looks like she was meant to have a custom outfit in changes.ini.~
![swkotor 2019-05-15 12-48-23-04](https://user-images.githubusercontent.com/46898647/57925591-c4847580-7876-11e9-829f-5d155f816807.png)
~2. Hallway blaster is back. I thought we fixed that?~
![swkotor 2019-05-15 12-56-43-89](https://user-images.githubusercontent.com/46898647/57925666-06adb700-7877-11e9-8d99-2b8e3888bb46.png)
3. It's possible to interact with a transformer near the turbolift to the governor's office.
![swkotor 2019-05-15 13-01-49-51](https://user-images.githubusercontent.com/46898647/57925676-0ca39800-7877-11e9-9a4f-29cd02fd7a16.png)

**Comment by DarthParametric** [comment]
I originally gave the receptionist a custom black set of commoner clothes, but I think I took that out. There's not really much else to be done with her I don't think. Originally she wore combat armour.

That blaster is one of SithSpector's. He breaks it again because he adds a model to one of the nulls. Same as the one that spawns under the Hawk in the Tatooine hangar bay. I made some fixes for it, but he never responded about my War Droid fix, so I didn't bother telling him about those.
